code-challenges.test.js

```diff
@@ -20,8 +20,29 @@ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
-// b) Create the function that makes the test pass.
+describe("constColor", () => {
+    const colors1 = ["yellow", "blue", "pink", "green"]
+    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+     it("takes in an array, removes the first item from array, shuffles remaining content.", () => {
+      expect(constColor(colors1)).toEqual("purple", "blue", "green", "yellow", "pink")
+      expect(constColor(colors2)).toEqual("saffron", "aquamarine", "periwinkle", "indigo", "ochre")
+     })
+     })
+
+//b) Create the function that makes the test pass.
+// Pseudo code:
+//  function color  to the array
+//  let array slice
+//  let array shuffle, sort by math.random() -0.5"
+// Return the array shuffled.
```

Comment:
Your code and test look great, I would like to see your pseudocode be more descriptive on what each thing is doing.  Ex: Create a variable that will remove the first instance in the array (slice)